src/datumaro/plugins/data_formats/widerface.py

```diff
@@ -157,7 +157,7 @@ def _load_items(self, path):
                                     attributes[attr] = bbox_list[i]
                             i += 1
 
-                    annotations.append(
+                    items[item_id].annotations.append(
```

Comment:
Why only Widerface is affected by this change?

Reply:
At this line, `annotations: List` is created
https://github.com/openvinotoolkit/datumaro/blob/072c8a80398291a139f1f2739ae7dcd88b52ca4b/src/datumaro/plugins/data_formats/widerface.py#L111
Then, this `annotations` is used to construct `DatasetItem` here
https://github.com/openvinotoolkit/datumaro/blob/072c8a80398291a139f1f2739ae7dcd88b52ca4b/src/datumaro/plugins/data_formats/widerface.py#L123-L128
However, in the following lines, `Bbox` is pushed into `annotations`, not `items[item_id].annotations`,
https://github.com/openvinotoolkit/datumaro/blob/072c8a80398291a139f1f2739ae7dcd88b52ca4b/src/datumaro/plugins/data_formats/widerface.py#L160-L169

Previously, it should work because `id(annotations) == id(items[item_id].annotations)` (same Python object). However, after this PR change, 
```python
annotations: List
items[item_id].annotations: Annotations
=>
id(annotations) != id(items[item_id].annotations)
```
, so that `widerface.py#L160-L169` above will be not working.